modules/installation-bare-metal-config-yaml.adoc

```diff
@@ -178,7 +178,7 @@ ifndef::ibm-z,ibm-z-kvm[]
 endif::ibm-z,ibm-z-kvm[]
 endif::openshift-origin[]
 ifdef::openshift-origin[]
-additionalTrustBundle: | <13>
+additionalTrustBundle: | <14>
```

Comment:
This `adoc` file always trips me up 😖 can you clarify why the preview shows this as callout 15, but here in the diff it's callout 14?

Reply:
My bad, this applies to the OKD docs since it's in `ifdef::openshift-origin[]` conditional but I neglected to upload a preview of the OKD docs. Here's the live version: 

https://docs.okd.io/4.12/installing/installing_bare_metal/installing-restricted-networks-bare-metal.html#installation-bare-metal-config-yaml_installing-restricted-networks-bare-metal

"sshKey" and "additionalTrustBundle" both have <13> in the live docs so this will fix that.

Reply:
gotcha, makes sense. Thanks for explaining! 😎